src/Schema/Grammar.php

```diff
@@ -25,6 +27,61 @@ public function __construct()
         $this->addSingleStoreModifiers();
     }
 
+    /**
+     * Compile a change column command into a series of SQL statements.
+     *
+     * @param  \Illuminate\Database\Schema\Blueprint  $blueprint
+     * @param  \Illuminate\Support\Fluent  $command
+     * @param  \Illuminate\Database\Connection  $connection
+     * @return array|string
+     *
+     * @throws \RuntimeException
+     */
+    public function compileChange(Blueprint $blueprint, Fluent $command, Connection $connection)
+    {
+        if (version_compare(Application::VERSION, '10.0', '<')) {
+            throw new LogicException('This database driver does not support modifying columns on Laravel < 10.0.');
```

Comment:
What is the reason, why this is not supported for older versions of Laravel?

Reply:
Older Laravel versions doesn't support native column modifying, you may check:
* laravel/framework#45487
* laravel/framework#48864

Laravel was using `doctrine/dbal` before this version and I'm quit sure that it is going to be a problem for your package as Doctrine DBAL doesn't support Singlestore. But you may add tests to see if it fallbacks to MySQL correctly or not.

Reply:
in the latest commit 9777d8a I removed this and added test to see if it works with lower Laravel version + Doctrine DBAL

Reply:
Thanks!

Reply:
OK the tests shows that we can't use doctrine: 

> SQLSTATE[HY000]: General error: 1706 Feature 'ALTER TABLE CHANGE to modify column type' is not supported by SingleStore. Use MODIFY to change a column's type. (Connection: mysql, SQL: ALTER TABLE test CHANGE data data TEXT CHARACTER SET utf8mb4 DEFAULT NULL COLLATE `utf8mb4_unicode_ci`)